packages/edit-site/src/components/header/index.js

```diff
@@ -79,64 +80,92 @@ export default function Header( {
 	const displayBlockToolbar =
 		! isLargeViewport || deviceType !== 'Desktop' || hasFixedToolbar;
 
+	// Testing logic for hovered name...
+	const { hoveredBlockIds, getBlock, getBlockName } = useSelect(
+		( select ) => {
+			const { getHoveredBlocks, getBlock, getBlockName } = select(
+				'core/block-editor'
+			);
+
+			return {
+				hoveredBlockIds: getHoveredBlocks(),
+				getBlock,
+				getBlockName,
+			};
+		}
+	);
+	const nameToDisplay = useMemo( () => {
+		// Go through hovered blocks and see if one is of interest.
+		const templatePartBlockId = hoveredBlockIds.find(
+			( blockId ) => getBlockName( blockId ) === 'core/template-part'
+		);
+		const templatePartBlock = getBlock( templatePartBlockId );
+		return templatePartBlock?.attributes.slug || '';
```

Comment:
Will want to update this to get the actual Title from the entity if available.

Reply:
we can probably use `getBlockLabel` for this (we use it in other parts of the interface that say "header"). 

but we can tie that in once we are working with the other component :)